packages/navigator/src/hooks/useMounted.ts

```diff
@@ -0,0 +1,11 @@
+import { useEffect, useReducer } from 'react'
+import { nextTick } from '../helpers'
+
+export function useMounted() {
```

Comment:
```suggestion
export function useMountedAfterTick() {
```

이런 이름이 더 좋을거같아요 ㅎㅎ 함수 이름만 봐서는 useEffect에서 mount()를 바로 부를거같아서요.

Reply:
@tonyfromundefined 

useMounted 가 인자로 boolean 타입 afterTick 을 받는 것은 어떤가요?

Reply:
올 굿